tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py

```diff
@@ -40,19 +40,27 @@ def test_ftu_skip_tour(self):
         ssid = self.testvars['wifi']['ssid']
         psk = self.testvars['wifi'].get('psk')
         keymanagement = self.testvars['wifi'].get('keyManagement')
+        share_data_default = True
```

Comment:
I would agree if we weren't in a test.

To me, assertions/waits should be done against not computed values, except if we're testing functions like `copy`. Having a single point of change seems less important in a test rather than in production code, simplicity would be the motto. 

I don't really think that hiding the value behind a variable makes the wait more self-documented (but the failure message does). That is to say, if you don't have enough context with this function, you could end up like:
``` Python
        Wait(self.marionette).until(
            lambda m: self.ftu.is_share_data_enabled == share_data_default, # What is the expected value already here?
 message="Send Data UI state should match initial internal state")

# Whereas:
        Wait(self.marionette).until(
            lambda m: self.ftu.is_share_data_enabled == True,
 message="Send Data UI state should match initial internal state") # Okay, the initial state is supposed to be true here.
```

[Here's a blog post](http://googletesting.blogspot.fr/2014/07/testing-on-toilet-dont-put-logic-in.html) from the Google Testing blog with one example of the drawback of having factorized variables. 

What do you think?